src/frontend-samples/display-styles-sample/DisplayStylesApp.tsx

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import { DisplayStyle3dSettingsProps } from "@bentley/imodeljs-common";
+import { Viewport } from "@bentley/imodeljs-frontend";
+import SampleApp from "common/SampleApp";
+import "common/samples-common.scss";
+import * as React from "react";
+import DisplayStylesUI from "./DisplayStylesUI";
+import { displayStyles } from "./Styles";
+
+/** This sample highlights the ability to override the current display settings with a JObject or
+ * JSON describing the desired settings. All the descriptions are in the Styles.ts file.
+ *
+ * We invite you to create your own settings using the "Custom" tag provided in Styles.ts.
+ * You can copy and paste parts of your favorite styles as a start.
+ *
+ * Only the features specificized by the style will be changed. This can be seen with the "Custom"
+ * style you can create. Use the "Merge with Custom" toggle to apply both the custom and the specified
+ * styles.
```

Comment:
Grammar:
Use the "Merge with Custom" toggle to apply both the custom and the specified style.